backend/src/main/scala/cromwell/backend/package.scala

```diff
@@ -35,6 +35,9 @@ package object backend {
                                   key: BackendJobDescriptorKey,
                                   inputs: Map[LocallyQualifiedName, WdlValue]) {
     val call = key.call
+
+    /** The abort functionality potentially becomes available after the job descriptor is created */
+    var abortFunction: Option[() => Unit] = None
```

Comment:
ToL:

I'm not saying change this, just responding to the comment:

> if we were to send the abort message there, it couldn't do JES specific abort functionality

Could one handle the message in `AsyncBackendJobExecutionActor.receive`, while still allowing `JesAsyncBackendJobExecutionActor` to override a handler with specific JES code?

```scala
def receive = {
  // ...
  case Abort => handleAbort()
  // ...
}

protected def handleAbort(): Unit = {}
```